djangocms_pageadmin/admin.py

```diff
@@ -427,9 +434,90 @@ def get_urls(self):
                 self.admin_site.admin_view(self.set_home_view),
                 name="{}_{}_set_home_content".format(*info),
             ),
+            re_path(
+                r'^export_csv/$',
+                self.admin_site.admin_view(self.export_to_csv),
+                name="{}_{}_export_csv".format(*info),
+            ),
         ]
         return new_urls + old_urls
 
+    def _format_export_datetime(self, date):
+        """
+        date: DateTime object
+        date_format: String, date time string format for strftime
+
+        Returns a formatted human readable date time string
+        """
+        import datetime
```

Comment:
Why is this imported within the method rather than at the top of the file as normal?

Reply:
My bad! Added it there while working on the code, moved to the top now